students/Ruohan/lesson04/mailroom_part2

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/env python3
```

Comment:
this looks like an error -- you should remove it from git:

`git rm mailroom_part2`